packages/playwright/src/program.ts

```diff
@@ -203,10 +203,13 @@ async function runTests(args: string[], opts: { [key: string]: any }) {
 
   const runner = new Runner(config);
   let status: FullResult['status'];
-  if (process.env.PWTEST_WATCH)
+  if (process.env.PWTEST_WATCH) {
```

Comment:
We should move this further up, similar to the line 164, without loading the config at all, and without involving `Runner`.